tpp/src/main/TokenClient.js

```diff
@@ -183,6 +185,7 @@ export class TokenClient extends Core {
         frequency: string,
         startDate: string,
         endDate: string,
+        destinations: Array<TransferDestination>,
```

Comment:
I don't think we should change this endpoint.
1. it is a breaking change.
2. compare to other token request build creation methods, it seems we prefer to let the tpp set destination with the builder methods instead. we should keep the behavior.

Reply:
1. If I'm not mistaken, in JS it is not breaking because you can 
2. The pattern is to put required fields in the constructor. It's required in standing order tokens but not transfer tokens because of the legacy flow where you could set transfer destinations on redemption. If you look at sdk-java, it has `List<TransferDestination>` in the constructor.